token/solana.tokenlist.json

```diff
@@ -0,0 +1,11650 @@
+{
```

Comment:
maybe can do this as a curl so we dont have to add a large json file we have to update in this repo